test/inductor/test_cudagraph_trees.py

```diff
@@ -102,6 +102,10 @@ def setUp(self):
             super().setUp()
             self.prev_enabled = config.triton.cudagraphs
             self.tapes_enabled = config.triton.cudagraph_trees
+            self.prev_slow = config.triton.slow_path_cudagraph_asserts
+            self.prev_fast = config.triton.fast_path_cudagraph_asserts
+            config.triton.fast_path_cudagraph_asserts = True
+            config.triton.slow_path_cudagraph_asserts = True
```

Comment:
Suggest using `config.patch({"triton.fast_path_cudagraph_asserts": True, ...})` with a `contextlib.ExitStack()` to simplify this